docs/developers/contributing/testing.md

```diff
@@ -191,62 +191,61 @@ See also [this paper on property-based testing in science](https://conference.sc
-
-1. If you need a `QApplication` to be running for your test, you can use the
-   [`qtbot`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot) fixture from `pytest-qt`
-
-    > note: fixtures in pytest can be a little mysterious, since it's not always
-    > clear where they are coming from.  In this case, using a pytest-qt fixture
-    > looks like this:
-
-    ```python
-    # just by putting `qtbot` in the list of arguments
-    # pytest-qt will start up an event loop for you
-    def test_something(qtbot):
-        ...
-    ```
-
-   `qtbot` provides a convenient
-   [`addWidget`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot.addWidget)
-   method that will ensure that the widget gets closed at the end of the test.
-   It *also* provides a whole bunch of other
-   convenient methods for interacting with your GUI tests (clicking, waiting
-   signals, etc...).  See the [`qtbot` docs](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot) for details.
-
-    ```python
-    # the qtbot provides convenience methods like addWidget
-    def test_something_else(qtbot):
-        widget = QWidget()
-        qtbot.addWidget(widget)  # tell qtbot to clean this widget later
-        ...
-    ```
-
-2. When writing a test that requires a `napari.Viewer` object, we provide a
-   [pytest fixture](https://docs.pytest.org/en/stable/explanation/fixtures.html) called
-   `make_napari_viewer` that will take care of creating a viewer and cleaning up
-   at the end of the test.  When using this function, it is **not** necessary to
-   use a `qtbot` fixture, nor should you do any additional cleanup (such as
-   using `qtbot.addWidget` or calling `viewer.close()`) at the end of the test.
-   Duplicate cleanup may cause an error.  Use the fixture as follows:
-
-    ```python
-    # the make_napari_viewer fixture is defined in napari/utils/_testsupport.py
-    def test_something_with_a_viewer(make_napari_viewer):
-        # make_napari_viewer takes any keyword arguments that napari.Viewer() takes
-        viewer = make_napari_viewer()
-
-        # do stuff with the viewer, no qtbot or viewer.close() methods needed.
-        ...
-    ```
-
-> If you're curious to see the actual `make_napari_viewer` fixture definition, it's
-> in [`napari/utils/_testsupport.py`](https://github.com/napari/napari/blob/main/napari/utils/_testsupport.py).
+loop or a {class}`~napari.Viewer` is required.  The important thing is that any widgets
+you create during testing need to be cleaned up at the end of each test. We thus
+recommend that you use the following fixtures when needing a widget or
+{class}`~napari.Viewer` in a test.
+
+#### qtbot
+
+If you need a `QApplication` to be running for your test, you can use the
+[`qtbot`](https://pytest-qt.readthedocs.io/en/latest/reference.html#pytestqt.qtbot.QtBot) fixture from `pytest-qt`
+
+```{admonition}
```

Comment:
```suggestion
```{note}
```

A generic requires a title I think?

Reply:
Currently it looks like:
<img width="645" alt="image" src="https://github.com/napari/docs/assets/76622105/09d42a15-ff39-4dec-b11e-8bdbb84424a4">